metabox/metabox/main.py

```diff
@@ -61,6 +61,9 @@ def main():
     parser.add_argument(
         '--do-not-dispose', action='store_true',
         help="Do not delete LXD containers after the run")
+    parser.add_argument(
+        '--redeploy-existing', action='store_true',
+        help="Re-Deploy and install existing containers")
```

Comment:
TBH I don't know what do you mean to install a container.

Reply:
Can be made better, this means copy and install everything that needs to be copied/installed (providers + checkbox). Do you have a better description?